Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins/jenkins:2.176.2
+FROM jenkins/jenkins:2.60.3
```

Comment:
nope :)

Reply:
Actually, we were pretty much up to date. 2.176.3 is the latest, see https://jenkins.io/download/

Jenkins (like Linux) versioning is not based on the first digit, i.e. 60 < 176